lib/stalePullRequest.js

```diff
@@ -67,11 +67,17 @@ stalePullRequest._processRepository = function (repositoryName, repositorySettin
 
 stalePullRequest._processPullRequest = function (pullRequest, repositorySettings) {
     var commentsUrl = pullRequest.comments_url + '?sort=updated';
+    var commitsUrl = pullRequest.commits_url;
+    var daysSinceLastCommit;
 
     function processComments(commentsJsonResponse) {
         var lastComment = commentsJsonResponse[commentsJsonResponse.length - 1];
```

Comment:
Should we innclude a check here for that 0 comment edge case?

Reply:
It's accounted for in line 75:

```
var lastComment = commentsJsonResponse[commentsJsonResponse.length - 1];
var pullRequestDate = Cesium.defined(lastComment) ? lastComment.updated_at : pullRequest.updated_at;
```

Were you thinking of doing it a different way?

Reply:
But if commentsJsonResponse.length is 0, won't this index be out of bounds? Shouldn't we check for that instead?

Reply:
Yeah, I can move that check one line up. I think it'll be better for readability that way too.